deploy/helm/templates/rbac/clusterrole_binding.yaml

```diff
@@ -29,4 +29,21 @@ subjects:
     name: {{ include "kubeblocks.addonSAName" . }}
     namespace: {{ .Release.Namespace }}
 {{- end }}
+{{- if .Values.dataProtection.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
For dataprotection workers in a namespace, they should only have the permission to access the resource in this namespace, do not have permission to access other namespace resource, so we should create a RoleBinding for every namespace and use it. Ignore to use a ClusterRoleBinding.

Reply:
removed.